Makefile

```diff
@@ -0,0 +1,123 @@
+## Haiku Generic Makefile v2.6 ##
+
+## Fill in this file to specify the project being created, and the referenced
+## Makefile-Engine will do all of the hard work for you. This handles any
+## architecture of Haiku.
+
+# The name of the binary.
+NAME = BeApp
```

Comment:
@bach5000: With this tiny change the patchset will be okay:
```diff
-NAME = BeApp
+NAME = BePhoneBook
```

Reply:
@fbrosson Done! Just need to convert rsrc to rdef.